playlist.c

```diff
@@ -2769,6 +2780,55 @@ void playlist_get_db_name(playlist_t *playlist, size_t idx,
    }
 }
 
+core_info_t *playlist_entry_get_core(const struct playlist_entry* entry)
+{
+   core_info_ctx_find_t core_info;
+
+   core_info.inf = NULL;
+   core_info.path = entry->core_path;
+
+   if (!core_info_find(&core_info))
+      return NULL;
+
+   if (strcmp(core_info.inf->path, entry->core_path))
+   {
+      RARCH_LOG("[CORE] Found core '%s' path '%s' does not match requested path '%s'", core_info.inf->display_name, core_info.inf->path, entry->core_path);
+   }
+
+   return core_info.inf;
+}
+
+core_info_t *playlist_get_default_core(playlist_t* playlist)
+{
+   core_info_ctx_find_t core_info;
+   const char* default_core_path =
+      playlist_get_default_core_path(playlist);
+   const char* default_core_name =
+      playlist_get_default_core_path(playlist);
```

Comment:
This should be `playlist_get_default_core_name()`

Reply:
I removed playlist_get_default_core_name() because it was initially used to search core by display name.